windows-setup/package.sh

```diff
@@ -20,6 +20,7 @@ mkdir "$setup_dir"/lib
 
 echo "copy installed files"
 (cd ../build && cmake .. -DCMAKE_INSTALL_PREFIX= && DESTDIR=../windows-setup/"$setup_dir" cmake --build . --target install)
+copy xournalpp.pbd "$setup_dir"/bin/xournalpp.pbd
```

Comment:
This should be `cp`, right?
```suggestion
cp xournalpp.pbd "$setup_dir"/bin/xournalpp.pbd
```